README.md

```diff
@@ -8,14 +8,22 @@ The CMakeLists.txt uses 3rd party packages from /cvmfs which are identical to th
 code is needed to build the code from this repository. CMS header files and shared libraries are needed if data files containing CMS data are being read.
 
 To build
-1. git clone ....
-1. cd root_serialization
-1. mkdir build
-1. cd build
-1. cmake .. -DZSTD_DIR=path_to_zstd_install -DTBB_DIR=path_to_tbb_install -DLZ4_DIR=path_to_lz4_install -DROOT_DIR=path_to_root_install
-1. make
 
-This will create the executable `threaded_io_test`.
+```console
+$ git clone <root_serialization URL>
+$ mkdir build-root-serialization
+$ cd build-root-serialization
+$ cmake <path to root_serialization> \
+  -DROOT_DIR=$ROOTSYS/cmake \
```

Comment:
This assumes ROOT's runtime has already been setup. That hasn't been the case when I do the build.

Reply:
Adjusted.